src/sample/SQL/CAT SQL.sql

```diff
@@ -0,0 +1,83 @@
+CREATE DATABASE COURSE_REGISTER;
+USE COURSE_REGISTER;
+
+CREATE TABLE COURSE(
+ COURSE_ID CHAR(6) NOT NULL,
+ COURSE_NAME VARCHAR(100) NOT NULL,
+ PRIMARY KEY(COURSE_ID)
+ );
+ 
+ CREATE TABLE STUDENT(
+  MATRIC_NUMBER CHAR(6) NOT NULL,
+  STU_NAME VARCHAR(30) NOT NULL,
+  STU_IC_NUM CHAR(5) NOT NULL,
+  ACADDEMIC_STATUS VARCHAR(8) NOT NULL,
+  SEMESTER_REGISTERED CHAR(3) NOT NULL,
+  STU_CGPA DECIMAL(3,2) NOT NULL,
+  STU_YEAR CHAR(1) NOT NULL,
+  STU_SCHOOL VARCHAR(50) NOT NULL,
+  STU_CAMPUS VARCHAR(50) NOT NULL,
+  STU_PROGRAMME VARCHAR (50) NOT NULL,
+  PRIMARY KEY(MATRIC_NUMBER)
+  );
+  
+  CREATE TABLE LECTURER(
+   STAFF_ID CHAR(6) NOT NULL,
+   LECT_NAME VARCHAR(30) NOT NULL,
+   LECT_IC_NUMBER CHAR(5) NOT NULL,
+   LECT_STATUS VARCHAR(8) NOT NULL,
```

Comment:
Why Status is Varchar?

Reply:
status is varchar because there are 2 status - active and inactive